src/core/Akka/IO/UdpConnectedManager.cs

```diff
@@ -34,7 +34,7 @@ protected override bool Receive(object message)
             {
                 case UdpConnected.Connect connect:
                     {
-                        var commander = Sender;
+                        var commander = Sender; // NOTE: Aaronontheweb (9/1/2017) this should probably be the Handler...
```

Comment:
This seems like a bug to me; the `.Handler` property on the `Connect` message is never actually used... But changing it now would be a breaking change for anyone who has ever used this feature. Decided to just take note of it and punt on it until a later time in the interests of fixing sending behavior first.